apps/src/templates/progress/NewProgressBubble.jsx

```diff
@@ -142,10 +133,10 @@ const NewProgressBubble = React.createClass({
     let bubble = (
       <div
         style={{
-          display: 'inline-block',
           // two pixles on each side for border, 2 pixels on each side for margin
           width: (smallBubble ? SMALL_DOT_SIZE : DOT_SIZE) + 8,
-          textAlign: 'center',
+          display: 'flex',
+          justifyContent: 'center'
```

Comment:
What do we get out of using `flex` and `justify-content: center` on this div which only contains one child div, which is in turn `flex` in the same direction and with `justify-content: center`?